pkg/uroot/binary.go

```diff
@@ -13,8 +13,11 @@ import (
 
 // BinaryBuild builds all given packages as separate binaries and includes them
 // in the archive.
-func BinaryBuild(opts BuildOpts) (ArchiveFiles, error) {
-	af := NewArchiveFiles()
+func BinaryBuild(af ArchiveFiles, opts BuildOpts) error {
```

Comment:
Why do I need this again? I get this effect today with just and -extra argument?

Reply:
I just added it for the convenience of not having to build binaries yourself and then including them manually. Just intended as a test mode.